README.md

```diff
@@ -44,17 +46,24 @@ the distributed effort. But you can still play, especially if you are patient.
 
 ### Windows
 
-Head to the Github releases page at https://github.com/gcp/leela-zero/releases,
-download the latest release, unzip, and launch autogtp.exe. It will connect to
-the server automatically and do its work in the background, uploading results
+- Easiest way (recommended) : 
+If you want to run the latest stable version ("master" branch), head to the Github releases page at https://github.com/gcp/leela-zero/releases, download the latest release for windows, unzip, and launch autogtp.exe.
```

Comment:
People should run the master version unless they are confident they know what they're doing, so there's no reason to mention this. All the releases are for windows, again, no reason to mention this, it can only cause more confusion.